flowetl/tests/integration/test_qa.py

```
@@ -6,14 +6,14 @@
 
 @pytest.mark.parametrize("cdr_type", ["calls", "sms", "mds", "topups"])
-def test_added_rows(cdr_type, flowdb_transaction, jinja_env):
+def test_added_rows(cdr_type, flowdb_transaction, jinja_env_final_qa):
     create_sql = f"""CREATE TABLE IF NOT EXISTS events.{cdr_type}_20160101 (LIKE events.{cdr_type});"""
     insert_sql = f"""INSERT INTO events.{cdr_type}_20160101(datetime, msisdn, location_id) VALUES 
             ('2016-01-01 00:01:00'::timestamptz, '{"A" * 64}', '{"B" * 64}'), 
             ('2016-01-01 00:01:00'::timestamptz, '{"A" * 64}', '{"B" * 64}')"""
     flowdb_transaction.execute(create_sql)
     flowdb_transaction.execute(insert_sql)
-    check_sql = jinja_env.get_template("count_added_rows.sql").render(
+    check_sql = jinja_env_final_qa.get_template(f"count_added_rows.sql").render(
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)